.github/workflows/tests-commit.yaml

```diff
@@ -0,0 +1,37 @@
+name: Commit Tests
+on: # Run on all commit pushes to all feature branches.
```

Comment:
This runs on all commits regardless of whether the branch is in a PR?

Reply:
yes